cdc/sink/ddlsink/mysql/mysql_ddl_sink.go

```diff
@@ -247,3 +250,46 @@ func (m *DDLSink) Close() {
 		}
 	}
 }
+
+// asyncExecAddIndexDDLIfTimeout executes ddl in async mode.
+func (m *DDLSink) asyncExecAddIndexDDLIfTimeout(ctx context.Context, ddl *model.DDLEvent) error {
```

Comment:
what will happen if return error ?

Reply:
cdc will retry this changefeed

Reply:
if the DDL execution time is more than 1 second,  this function always returns a nil.